cookiecutter.json

```diff
@@ -0,0 +1,17 @@
+{
+  "snippet_name": "MySnippet",
+  "project_name": "NetSquid-{{ cookiecutter.snippet_name }}",
```

Comment:
It should be clear when filling in snippet_name that we will prepend "Netsquid-" to it, otherwise user might try prepending it themselves e.g. a prompt something like this:
```
snippet_name [NetSquid-{snippet_name}]: 
```

Also seems there is one too many options here e.g. what is difference between snippet_name and project_name? Can we drop project_name, and just use NetSquid-{snippet_name} to fill in the slug and directory (note: linux -> directory, windows -> folder)?

Reply:
It would be nice if one could have variables which default without showing this to the user. This is currently an open issue in `cookiecutter`, see for example https://github.com/cookiecutter/cookiecutter/issues/1115

At the moment one can just press Enter three times, which might be fine? I wrote a `pre_gen_script` which checks the names given to make sure that the convention is followed.

Reply:
What about wrapping the cookiecutter command with a shell script that prints a few lines of explanation text before the questions start? The script could take as a single input the destination directory

Reply:
This is now done throught the script `make.sh`.